py_inspect.py

```diff
@@ -629,6 +636,15 @@ def closeEvent(self, event):
 
     # Actions
 
+    # TODO use hooks + find in tree + fix draw
+    def __lookForMouse(self):
+        desktop = pywinauto.Desktop(backend='uia')
+        while True:
+            time.sleep(0.1)
+            if self.mouse.isChecked():
+                x, y = win32api.GetCursorPos()
+                desktop.from_point(x, y).draw_outline()
```

Comment:
I'd prefer to draw the rectangle only by manual pressing some key on a keyboard.

Reply:
added hotkey "s" for start searching and hotkey "d" for enable\disable drawing